ios/FluentUI.Demo/FluentUI.Demo/Demos/TooltipDemoController.swift

```diff
@@ -131,3 +131,51 @@ extension TooltipDemoController: TwoLineTitleViewDelegate {
         present(alert, animated: true)
     }
 }
+
+extension TooltipDemoController: DemoAppearanceDelegate {
+    func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return
+        }
+
+        fluentTheme.register(tokenSetType: TooltipTokenSet.self,
+                             tokenSet: isOverrideEnabled ? themeWideOverrideTooltipTokens : nil)
+    }
+
+    func perControlOverrideDidChange(isOverrideEnabled: Bool) {
+        Tooltip.shared.tokenSet.replaceAllOverrides(with: isOverrideEnabled ? perControlOverrideTooltipTokens : nil)
+    }
+
+    func isThemeWideOverrideApplied() -> Bool {
+        return self.view.window?.fluentTheme.tokens(for: TooltipTokenSet.self) != nil
+    }
+
+    // MARK: - Custom tokens
+    private var themeWideOverrideTooltipTokens: [TooltipTokenSet.Tokens: ControlTokenValue] {
+        return [
+            .tooltipColor: .dynamicColor {
+                // "Berry"
+                return DynamicColor(light: GlobalTokens.sharedColors(.berry, .tint50),
+                                    dark: GlobalTokens.sharedColors(.berry, .shade40))
+            }
+        ]
+    }
+
+    private var perControlOverrideTooltipTokens: [TooltipTokenSet.Tokens: ControlTokenValue] {
```

Comment:
This combination of colors looks like it might not satisfy accessibility contrast requirements. Have you checked it with the colour contrast analyser? Otherwise we might get accessibility bugs filed on us!

Reply:
Even if it's just demo colors to test the override? I just used the same colors Sophia used for the TableViewCell demo

Reply:
Gotcha, good catch. I'll update it! Thanks 😊